src/lib.rs

```diff
@@ -203,20 +203,20 @@ pub fn is_blob_uri(uri: &str) -> Result<bool> {
 /// Check whether the given URI is a multiserver address URI.
 pub fn is_multiserver_uri(uri: &str) -> Result<bool> {
```

Comment:
Could this function now return a `bool` instead of `Result<bool>`?

Reply:
Good question.

Since `Url::parse(uri)` (L205) returns a `Result<Url, ParseError>` ([docs](https://docs.rs/url/2.2.2/url/struct.Url.html#method.parse)), I think it's best to keep the `Result<bool>`. Then any parse error will be returned as an `Err`.

Reply:
Ah, I see, the `?` unwrap on 205 means that the Result could be an error. Cool cool